src/project-css/app/Http/Controllers/TableController.php

```diff
@@ -400,6 +400,13 @@ public function finalize(Request $request){
             $table->bigInteger($curColNme)->default(0);
           }
         }
+
+        if(str_is($curColType,'text')){
```

Comment:
Can an error message be given to the user if providing the wrong datatype in these instances?  If not when nothing happens that the user expects it could cause confusion.